test/runner/requirements/constraints.txt

```diff
@@ -18,3 +18,5 @@ requests-ntlm >= 1.1.0 # message encryption support
 requests-credssp >= 0.1.0 # message encryption support
 voluptuous >= 0.11.0 # Schema recursion via Self
 openshift >= 0.6.2 # merge_type support
+jinja2 >=2.7 # 'select' filter appeared in this version
```

Comment:
Actually given #32738 (pallets/jinja#708) we might need to start requiring minimum version to be 2.10 at some point.

cc @jctanner

Reply:
P.S. Without `select` the test playbook appears purely readable and I want to check in something supportable.

@bcoca do you have any other ways of doing the same in mind?

Reply:
If the minimum requirements change, then [requirements.txt](https://github.com/ansible/ansible/blob/devel/requirements.txt) needs to be updated first. However, since https://github.com/ansible/ansible/pull/32738 is optional the requirements should not need to be updated.

Reply:
Can we just exclude this test from running on CentOS instead of rewriting? It doesn't really test anything OS-specific.

Reply:
There are several ways you could set `until_tempfile_path_var_names` without using `select`.

One option would be as part of the initial `set_fact` loop:

```yaml
- set_fact:
    "until_tempfile_path_{{ idx }}": "{{ tmp_file.stdout }}"
    until_tempfile_path_var_names: "{{ [ 'until_tempfile_path_%s' % idx ] + until_tempfile_path_var_names | default([]) }}"
  loop: "{{ tempfilepaths.results }}"
  loop_control:
    index_var: idx
    loop_var: tmp_file
```

Another would be to just manually list out the variable names:

```yaml
- set_fact:
    until_tempfile_path_var_names:
      - until_tempfile_path_3 
      - until_tempfile_path_2 
      - until_tempfile_path_1 
      - until_tempfile_path_0
 ```

There are likely others I haven't thought of.

Reply:
Nice, I'll go for it then.